src/components/common/CustomCard.js

```diff
@@ -47,6 +47,8 @@ function CustomCard(props) {
           <SvgIcon
             sx={{
               ...sx,
```

Comment:
Could you move the sx beneath the height and width? That way we can override the default height/width if we need to, some icons need to be smaller/larger depending on the case

Reply:
I see sure, making the change now.